types/coin_type.go

```diff
@@ -234,3 +234,19 @@ func GetCoinName(coinStr string) (coinName string, err error) {
 	coinName = strings.ToLower(coinName)
 	return coinName, nil
 }
+
+func GetCoinType(coinName string) (CoinType, error) {
+	var coinType CoinType
```

Comment:
Why not just add coinName empty checking in `GetCoinName`?

Reply:
I debug GetCoinName and find that If coinName is empty, the GetCoinName func will return err